tests/unittests/data/io/test_pawian.py

```diff
@@ -0,0 +1,116 @@
+"""Test :py:mod:`pycompwa.data.import.pawian`."""
```

Comment:
Since its also tested in ComPWA, this is also not needed here

Reply:
https://github.com/ComPWA/pycompwa/pull/78#discussion_r408925715